repository_service_tuf/cli/artifact/repository.py

```diff
@@ -0,0 +1,278 @@
+# SPDX-FileCopyrightText: 2023 VMware Inc
+#
+# SPDX-License-Identifier: MIT
+
+
+import base64
+
+from click import Context
+from dynaconf.loaders.yaml_loader import write  # type: ignore
+
+from repository_service_tuf.cli import click, console
+from repository_service_tuf.cli.artifact import artifact
+
+
+def write_config(settings_path, settings_data, merge=True):
```

Comment:
Maybe the default could be `merge=False` as everywhere you call it with `merge=False`. :))